app/controllers/admin/index.js

```diff
@@ -0,0 +1,23 @@
+import classic from 'ember-classic-decorator';
+import { action } from '@ember/object';
+import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+
+@classic
+export default class IndexController extends Controller {
+
+@tracked
+frontendVersion = '';
+
+@tracked
+serverVersion = '';
+
+@action
+async setup() {
+  const fVersion = await this.loader.load('https://api.github.com/repos/fossasia/open-event-frontend/git/refs/heads/development', { isExternal: true });
```

Comment:
Version should be shown for the master branch.
@mariobehling what do you suggest?

Reply:
```suggestion
  const fVersion = await this.loader.load('https://api.github.com/repos/fossasia/open-event-frontend/git/refs/heads/master', { isExternal: true });
```